taglib/riff/wav/wavfile.h

```diff
@@ -126,6 +127,11 @@ namespace TagLib {
         Info::Tag *InfoTag() const;
 
         /*!
+         * Returns the Acid loop information for this file
+         */
+        AcidInfo *acidInfo() const;
```

Comment:
Bummer, I was unaware it was incomplete ... I may have bitten off more then I can chew on this one!